### DISCUSSION TITLE
Adding flet to Chatgpt Plugin

### DISCUSSION THREAD

**Discussion by mswoff** [flet-dev/flet#1839] [Show and tell]
I wanted to let you know that users of our Code Library Search Chatgpt plugin have requested we support flet. Unless we hear otherwise from you, we’ll add support for flet (free for open-source projects and no action required from your team).

The plugin let's users get library specific coding help by analyzing your library's codebase and then uses GPT-4 to write and debug code specific to the library and version.

If you want to get in touch about any concerns or offering AI based support directly on your forum/website, please email me at mason@salv.ai

p.s. we may charge in the future for open source projects supported by large companies

**Comment by FeodorFitsner** [comment]
Thank you! Let me know if you need anything from us.
